src/System.CommandLine/ArgumentArity.cs

```diff
@@ -7,10 +7,21 @@
 
 namespace System.CommandLine
 {
+    /// <inheritdoc />
     public class ArgumentArity : IArgumentArity
     {
+        /// <summary>
+        /// The overall maximum number of values that can be provided to any argument.
+        /// </summary>
```

Comment:
I think this should actually be an internal `const` and not require a doc comment.

If you'd like to make the change, feel free to just specify it literally in the tests currently referencing it.

Reply:
OK just to confirm, change the modifier to `internal` and remove the XML comment.
Since that makes it inaccessible to the tests, you want to change this to a literal (12 places in 2 classes)?
``` csharp
[InlineData(0, ArgumentArity.MaximumArity, typeof(string[]))]
``` 
to
``` csharp
[InlineData(0, 100_000, typeof(string[]))]
```
or maybe better to add a const to each of the tests and keep things a bit DRY?

``` csharp
const int MaximumArity = 100_000;


[InlineData(0, MaximumArity, typeof(string[]))]
```

Reply:
On second thought, I guess you mean just remove the `public` modifier and make it a private const to the class?

Reply:
Yes. I don't think DRY in test code is as important as keeping the library API surface smaller, and in the case of these tests, the specific value isn't very important. Basically, this is a large number that's lower than `int.MaxValue` so we don't overflow.

Reply:
Done